sysdump/sysdump.go

```diff
@@ -772,27 +777,27 @@ func (c *Collector) Run() error {
 		if t.CreatesSubtasks {
 			c.subtasksWg.Add(1)
 		}
-		if err := c.pool.Submit(fmt.Sprintf("[%d] %s", i, t.Description), func(ctx context.Context) error {
+		if err := c.Pool.Submit(fmt.Sprintf("[%d] %s", i, t.Description), func(ctx context.Context) error {
 			if t.CreatesSubtasks {
 				defer c.subtasksWg.Done()
 			}
 			c.logTask(t.Description)
 			defer c.logDebug("Finished %q", t.Description)
 			return t.Task(ctx)
 		}); err != nil {
-			return fmt.Errorf("failed to submit task to the worker pool: %w", err)
+			return fmt.Errorf("failed to submit task to the worker Pool: %w", err)
```

Comment:
Nit: I don't think we want to change `s/pool/Pool` here and in other error and log messages.